governance/xc_admin/packages/xc_admin_common/src/programs/types.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Represents the different Pyth programs supported by the application.
+ */
+export enum ProgramType {
+  /**
+   * Original Pyth oracle program
+   */
+  PYTH_CORE = "PYTH_CORE",
+
+  /**
+   * Next-generation Pyth oracle program
+   */
+  PYTH_LAZER = "PYTH_LAZER",
+}
```

Comment:
I generally recommend against string enums, just use regular enums -- string enums don't really have any benefits over regular enums apart from automatic human-readable serialization, but that comes at a greater memory overhead cost and usually isn't actually useful -- enums should intentionally be opaque and if you want to map to clean names, use a record.